doc/swagger.v1.yml

```diff
@@ -55,6 +55,59 @@ paths:
             $ref: '#/definitions/Error401'
       security:
         - foodsoft_auth: ['all']
+
+  /financial_transactions:
+    get:
+      summary: financial transactions of the member's ordergroup
+      tags:
+        - 6. FinancialTransaction
+      parameters:
+        - $ref: '#/parameters/page'
```

Comment:
can we combine this two? they are use together more or less ever IMHO

Reply:
That _might_ be possible.
Don't you think it is clearer to list all parameters here, instead of having to lookup the definition to see what parameter names they contain?

Reply:
what's different to `Meta`? but i can live with both

Reply:
`Meta` is something that is returned, these parts are all described in models below. `parameters` are what you supply as part of the request.
I've tried various ways of structuring parametes in other API definitions, and settled on this flat list, which turned out to be the easiest to comprehend (and work with clients consuming the OpenAPI definition).